app/controllers/home_controller.rb

```diff
@@ -1,8 +1,17 @@
 class HomeController < ApplicationController
-  def index
+  def home
     @users = User.all
     @projects = Project.all
     @samples = Sample.all
-    @pipeline_outputs = PipelineOutput.all
+    @output_data = Hash.new
+
+    @pipeline_outputs = PipelineOutput.order("created_at DESC")
+
+    @pipeline_outputs.each do | output |
+      sample_info = Sample.find(output.sample_id)
```

Comment:
Consider using ActiveRecord associations for the PipelineOutput model
http://guides.rubyonrails.org/association_basics.html